com.google.AndroidStudio.appdata.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+	<id>com.google.AndroidStudio.desktop</id>
+	<metadata_license>CC-BY-SA-3.0</metadata_license>
```

Comment:
Any reason for not `CC0-1.0`?

Reply:
As above, it was derived from https://github.com/endlessm/android-studio-flatpak/blob/master/com.google.AndroidStudio.appdata.xml however since that is copyright my employer, I suspect we can fix that.